units/reauthorize_user.go

```diff
@@ -127,31 +127,27 @@ func (j *reauthorizationJob) Run(ctx context.Context) {
 	// Okta has expired, in which case they should be logged out, so that they
 	// are forced to log in to get a new refresh token.
 	if err != nil && strings.Contains(err.Error(), "invalid_grant") && strings.Contains(err.Error(), "The refresh token is invalid or expired.") {
-		grip.Debug(message.WrapError(err, message.Fields{
+		grip.Info(message.WrapError(err, message.Fields{
 			"message": "user's refresh token is invalid, logging them out",
 			"user":    j.UserID,
 			"job":     j.ID(),
 		}))
-		if err = user.ClearLoginCache(j.user, false); err != nil {
-			j.AddError(errors.Wrapf(err, "clearing login cache for user '%s'", j.UserID))
+		if err = user.ClearLoginCache(j.user); err != nil {
+			j.AddError(errors.Wrapf(err, "clearing login cache"))
 		}
 		return
 	}
 
-	if err != nil {
-		grip.Warning(message.WrapError(err, message.Fields{
-			"message": "could not reauthorize user",
+	j.AddRetryableError(errors.Wrap(err, "reauthorizing user"))
+
+	if err != nil && j.RetryInfo().GetRemainingAttempts() == 0 {
+		grip.Info(message.WrapError(err, message.Fields{
```

Comment:
Should this be `grip.Error`?

Reply:
Done. I don't know if this is worth it since it could increase the production error rate spuriously, but I don't expect this to log often so it's probably fine.